static/app/components/smartSearchBar/index.tsx

```diff
@@ -578,6 +578,13 @@ class SmartSearchBar extends React.Component<Props, State> {
     if (!this.searchInput.current) {
       return -1;
     }
+
+    // No cursor possition when the input loses focus. This is important for
+    // updating the search highlighers active state
+    if (this.searchInput.current !== document.activeElement) {
```

Comment:
Not sure if it applies but might you want to do `...current.contains(document.activeElement)` in case there is a tree inside the 'input'? Or will it always be accurate to just pick the single dom element?